src/Lib/Canvas.vala

```diff
@@ -108,7 +109,7 @@ public class Akira.Lib.Canvas : Goo.Canvas {
             // Create the image manager.
             var manager = new Lib.Managers.ImageManager (image, index);
             // Let the app know that we're adding image items.
-            window.event_bus.insert_item ("image");
+
```

Comment:
You moved the method but you forgot the comment.
I'll fix this, but try to be more aware of the code around and the structure you're touching.